test/utilities.js

```diff
@@ -1,6 +1,8 @@
-const waitForNBlocks = async function(numBlocks, authority) {
+
```

Comment:
Can we clean this up? `timeTravel` and `increaseTimeBy` should do the same thing (same for `timestamp` and `getTime`). If they are not the same thing their difference should be made clear.

Generally it would be nice if this kind of utility could be provided by a library. It feels like we create this type of helper in all our repos.

Reply:
Cleaned it up. Yeah, we also discussed this library approach in one of your weekly smart contracts.

Unfortunately, it never got any traction...